pep8.py

```diff
@@ -1846,7 +1846,15 @@ def check_files(self, paths=None):
         report.start()
         try:
             for path in paths:
-                if os.path.isdir(path):
+                if path == "-":
+                    # STDIN
+                    path = self.options.stdin_name
```

Comment:
Cool. @leorochael by the way, Flake8 3.0 beta releases support this for you. And it will be supported when 3.0 final is released as well. :)